libs/feature/editor/src/lib/components/record-form/form-field/form-field.component.html

```diff
@@ -1,10 +1,11 @@
-<ng-container *ngIf="model === 'licenses'">
+<!-- TEMPORARY - disabling the open data switch -->
+<!-- <ng-container *ngIf="model === 'licenses'">
   <gn-ui-form-field-open-data
     [value]="valueAsConstraints"
     (valueChange)="valueChange.emit($event)"
     (openDataChange)="isHidden = $event"
   ></gn-ui-form-field-open-data>
-</ng-container>
+</ng-container> -->
 <div class="flex flex-col h-full" *ngIf="!isHidden">
```

Comment:
I'm not sure if this ngIf is still needed, since the value of it was controlled by the gn-ui-form-field-open-data.
But it also does no harm I guess. Just food for thought.

Reply:
True, will remove it!